### PR TITLE
Feature/application

### PR DESCRIPTION
**Pull request by open-schnick**
:D

### CODE REVIEW

File: config.cfg [hunk 1]
```diff
@@ -4,4 +4,4 @@ db_user=root
 db_port=27017
 db_name=filefighter
 use_stable_versions=true
-db_password=
+db_password=mvAqaAAleIZWIOiPZWsQfnCsDiTvUbbmrgrzmaVZoTyGNagrLu
```
Comment (open-schnick) [inline config.cfg:7]: ups.

File: ffighter [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  exit 0
+fi
+
+echo "FileFighter Application $version."
+echo ""
+echo "Command could not be found."
+
+printUsage
+
+cd $oldwd
```
Comment (qvalentin) [inline ffighter:54]: Wrong var?

File: ffighter [hunk 3]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# changing directory to be able to use relative paths.
+starting_working_directory=$(pwd)
+rootDir="/home/open-schnick/Documents/projects/filefighter/ClientSetup/" # how do we get that?
```
Comment (qvalentin) [inline ffighter:5]: https://www.cyberciti.biz/faq/linux-unix-shell-script-find-out-in-which-directory-script-file-resides/

Comment (qvalentin) [reply ffighter:5]: https://www.golinuxcloud.com/get-script-name-get-script-path-shell-script/

Easier tutorial

**Review by qvalentin — APPROVED** [review]

**Comment by open-schnick** [comment]
Test it again pls @qvalentin

**Comment by qvalentin** [comment]
Yes it does add the folder where i cloned the repo correctly to the PATH

**Comment by open-schnick** [comment]
So i will upate the readme and merge

**Review by qvalentin — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -39,27 +39,76 @@ sudo snap install docker
...
+  install   - install the FileFighter application.
+  start     - start the services.
+  stop      - stop the services.
+  remove    - remove all services.
+```
+You can see all the available options to run with *ffighter*.  
+
+| Option      | Description |
+| :----:   |  :----:  |
+| status | This command shows you information about the status of the application, like install status, whether its running or not etc... |
```
Comment (qvalentin) [inline README.md:94]: installation status

**Review by qvalentin — APPROVED** [review]

**Pull request merged**